sage_codes/Point computation 

```diff
@@ -87,6 +87,8 @@ sage:         z += 1
 sage: prime_mod_sqrt(y,p)
 [5464794816676661649783249706827271879994893912039750480019443499440603127256,
 1772210760655600564189936856215722360862222447340157125982507438844851123733]
+/// The assumption from above is to take x, and not p-x as the postive value 
```

Comment:
This might change, at least since in Decaf it's said that a positive value is the one which relies on: [0, (L-1) /2]. 

Indeed, all of the positiveness checkings done in the Ristretto implementation follow this guideline.

What do you think?